keras_nlp/samplers/top_k_sampler_test.py

```diff
@@ -94,6 +94,21 @@ def next(prompt, state, index):
         output_ids = set(output[0].numpy())
         self.assertContainsSubset(output_ids, range(5))
 
+    def test_outputs_in_top_k_with_temperature(self):
```

Comment:
I wonder if as a very simple version of this we can just do `self.sampler = TopKSampler(k=5, temperature=1.0)` in setup above. The temp won't affect the output for top-k right?

Reply:
Yes, it does not affect the output, I have thereby removed `def test_outputs_in_top_k_with_temperature(self):`